internal/envoy/config/filters.go

```diff
@@ -95,6 +96,9 @@ func NewHTTPExternalAuthorizationFilter(upstreamHostname string, upstreamPort ui
 	authorization := &envoy_auth_v3.ExtAuthz{
 		Services:            services,
 		TransportApiVersion: envoy_config_core_v3.ApiVersion_V3,
+		StatusOnError: &envoy_type_v3.HttpStatus{
+			Code: envoy_type_v3.StatusCode_ServiceUnavailable,
+		},
```

Comment:
Great concise solution

Reply:
```
=== RUN   TestBasicAuthCheckSuite/TestForbidden
    basicauth_test.go:98: 
        	Error Trace:	basicauth_test.go:98
        	Error:      	Not equal: 
        	            	expected: 403
        	            	actual  : 503
        	Test:       	TestBasicAuthCheckSuite/TestForbidden
```
It seems to return a 503 when it should return a 403 when the auth service is up but the user is actually forbidden

Reply:
Locally it passes without a glitch. 

Is the CI acting up for some reason?

Reply:
I don't think CI errors are related to this PR see below
```
=== RUN   TestMockingSuite/TestEndpoint
    mocking_test.go:53: 
        	Error Trace:	mocking_test.go:53
        	Error:      	Received unexpected error:
        	            	Get "http://34.148.71.124/hello": dial tcp 34.148.71.124:80: connect: connection refused
        	Test:       	TestMockingSuite/TestEndpoint
    suite.go:63: test panicked: runtime error: invalid memory address or nil pointer dereference
```